SHMEM/isx.h

```diff
@@ -58,7 +58,7 @@ static void create_permutation_array();
 /*
  * Randomly shuffles a generic array
```

Comment:
Might be a good idea to update the comment to read "Randomly shuffles an array of PE ids"